ros/src/tl_detector/tl_detector.py

```diff
@@ -134,9 +197,9 @@ def process_traffic_lights(self):
         light = None
 
         # List of positions that correspond to the line to stop in front of for a given intersection
-        stop_line_positions = self.config['stop_line_positions']
+        #stop_line_positions = self.config['stop_line_positions']
         if(self.pose):
-            car_position = self.get_closest_waypoint(self.pose.pose)
+        	car_position = self.get_closest_waypoint(self.pose.pose)
```

Comment:
Although this isn't used now, may want to update the function arguments to match the definition.

Reply:
Yep - will fix this.